matlab/Ice/greeter/Greeter.ice

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) ZeroC, Inc.
+
+#pragma once
+
+module VisitorCenter
+{
+    /// Represents a simple greeter.
+    interface Greeter
+    {
+        /// Creates a personalized greeting.
+        /// @param name: The name of the person to greet.
+        /// @returns: The greeting.
```

Comment:
```suggestion
        /// @param name The name of the person to greet.
        /// @return The greeting.
```
This doc-comment is incorrect:
1) There shouldn't be any colon separators in Slice1-Ice-Slice because Doxygen doesn't support it.
2) `@returns` isn't a supported tag. It should be `@return`.
The parser is letting it through because it isn't enforcing that there's a space after `@return`, a bug.
I'll add this to the comment-validation issue.